server/lib/endpoint-users.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const GatewayEndpoint = require('./endpoint');
+
+class GatewayEndpointUsers extends GatewayEndpoint {
```

Comment:
This class is missing an `update` function which we'd need in Conversations, but is also where I've been considering whether to just verbs as our functions, e.g. `northstar.get,`northstar.post`,`northstar.put` vs `updateUser`, `createUser`, etc

Reply:
Working on this more, and we're better with separate endpoint functions as creating a Rogue post needs to send multipart/form-data. Will update the PR title and description when ready for a review, as we're looking to add the Rogue `GET /campaigns` endpoint into this lib to help deprecate Ashes 🔥